go.mod

```diff
@@ -8,6 +8,8 @@ exclude k8s.io/kubernetes v1.13.0
 // Client-go does not handle different versions of mergo due to some breaking changes - use the matching version
 replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.5
 
+replace istio.io/api => github.com/howardjohn/api v0.0.0-20220310185844-9962667afd9d
```

Comment:
I don't think you wanted to check this in

Reply:
Yeah I had a hold to fix this -- want to let tests run